SCClassLibrary/Common/Collections/extLazy.sc

```diff
@@ -0,0 +1,7 @@
++ LazyEnvir{
+	put { arg key, obj;
+		if(obj.isFunction){obj=obj.inEnvir(this.envir)};
```

Comment:
thanks! Could you keep the standard formatting (spaces between operators)? Normally, I wouldn't care, but for the common class library it is better.